salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/FMCMainDisplay.js

```diff
@@ -287,17 +298,53 @@ class FMCMainDisplay extends BaseAirliners {
                     this.setLine(template[tIndex][1], i, 1);
                     this.setLine(template[tIndex][2], i, 2);
                     this.setLine(template[tIndex][3], i, 3);
-                }
-                else {
+                } else {
                     this.setLine(template[tIndex][0], i, -1);
                 }
             }
         }
         if (template[13]) {
             this.setInOut(template[13][0]);
         }
-        SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex);
+        SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex).then();
```

Comment:
why the .then()?

Reply:
Typo.  I am new to this pull request stuff.  how do you recommend I send you an update to remove that from the pull request?

Reply:
switch to the "messages" branch on your fork, commit and push and the PR should update

Reply:
did it for you